geps/gep-91/index.md

```diff
@@ -8,11 +8,14 @@
 ## TLDR
 
 This GEP proposes a way to validate the TLS certificate presented by the frontend client to the server
-(Gateway Listener in this case) during a [TLS Handshake Protocol][].
+(Gateway in this case) during a [TLS Handshake Protocol][].
 
 ## Goals
 
-* Define an API field to specify the CA Certificate within the Gateway Listener configuration that can be used as a trust anchor to validate the certificates presented by the client. This use case has been highlighted in the [TLS Configuration GEP][] under segment 1 and in the [Gateway API TLS Use Cases][] document under point 7.
+* Define an API field to specify the CA Certificate within the Gateway configuration that can be used as a trust anchor to validate the certificates presented by the client.
+This use case has been highlighted in the [TLS Configuration GEP][] under segment 1 and in the [Gateway API TLS Use Cases][] document under point 7.
```

Comment:
We should fix these links while we're here.

Reply:
all these links work for me

Reply:
https://gateway-api.sigs.k8s.io/geps/gep-91/

Reply:
I hadn't seen this link syntax before, that's probably where the confusion comes from. (the links are in the Reference section at the end.)